### DISCUSSION TITLE
Add new layout for keybind cheatsheet

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1066] [Ideas]
Add new layout keybind key cheatsheet like this in nvim it is look good and also hightight with different color and scrolling is easy in thi type of layout @kRHYME7 
![240311_21h55m38s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/c7df8a46-1645-479a-9446-49269a6e5189)

**Comment by kRHYME7** [comment]
does rofi support this kind of layout? or any toolkit you can suggest to build on top of it?

Would you like to use it on the terminal only?

How about just searching/scrolling the keybindings on a single column list?

**Reply by mahaveergurjar** [reply]
I don't know rofi support this type of layout or not umm using vim or nvim ?? 
yeah terminal type also good 
No not in single column the cheat see like in three column or more the single column

**Reply by kRHYME7** [reply]
Ah, For now I will generate a markdown format so users can modify the output to their hearts desire.e.g using toolkit to render this or using the terminal.

I'm not familiar with nvim for this kind of work.

Also I don't really understand pretty well what is the implications of this. Having this in 2+ columns are fine. But we lack any toolkits on this repo.

 We only have rofi to provide a list  and we can conveniently search for certain keybindings using rofi.IMO this is fast and lightweight.

Also having this in columns is too unintuitive and needs lots of effort to render it. Categories for binds are quite scattered too.
It doesn't make sense for me at this moment.



Note that the script that provides Keybindings is generated on the fly base on your configurations and also generates categories which try to group the keybindings..
In the Image you provided the keybindings are already categorized. Which is not the script's capability.

**Reply by mahaveergurjar** [reply]
Ok then don't chnage then rofi is ok for this keybind hint
